src/opencl_bitlocker_fmt_plug.c

```diff
@@ -128,10 +128,13 @@ static struct fmt_main *self;
 #define SEED			1024
 
 static const char *warn[] = {
-        "xfer: ",  ", init: " , ", crypt: ", ", res xfer: "
+        "xfer: ",  ", ",  ", ",  ", ",  ", ",
+        ", ",  ", ",  ", ",  ", ",  ", ",
+        ", ",  ", ",  ", prepare: " ,  ", xfer: ",  ", crypt: ",
+        ", final: ",  ", res xfer: "
 };
 
-static int split_events[] = { 2, -1, -1 };
+static int split_events[] = { 14, 13, -1 };
```

Comment:
@claudioandre despite this split_events array, I get
```
xfer: 4.448us, 4.288us, 20.576us, 8.128us, 4.160us, 43.616ms, 3.776us, 35.744us, 3.744us, 2.944us, 2.784us, 3.232us, prepare: 308.384us, xfer: 4.064us, crypt: 16x71.767ms, final: 1.705ms, res xfer: 4.160us
```
although it should be
```
xfer: 4.448us, 4.288us, 20.576us, 8.128us, 4.160us, 43.616ms, 3.776us, 35.744us, 3.744us, 2.944us, 2.784us, 3.232us, prepare: 308.384us, xfer: 16x4.064us, crypt: 16x71.767ms, final: 1.705ms, res xfer: 4.160us
```
(the xfer right before loop crypt should be 16x too). Am I doing it wrong?